src/screen.rs

```diff
@@ -1361,11 +1367,9 @@ pub fn screen_force_clear_to_end() {
 /// Information about the layout of a prompt.
 #[derive(Clone, Debug, Default, Eq, PartialEq)]
 pub struct PromptLayout {
-    /// line breaks when rendering the prompt
-    pub line_breaks: Vec<usize>,
-    /// width of the longest line
-    pub max_line_width: usize,
```

Comment:
yeah that's the problem with using `pub` for testing,
we currently don't get "unused" warnings (though [that might change](https://internals.rust-lang.org/t/no-need-for-pub-crate-in-binary-crates/23607)), maybe we can write a clippy lint..